Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/StubColumnRenderer.cs

```diff
@@ -24,29 +24,39 @@
 
 namespace Remotion.ObjectBinding.Web.UnitTests.UI.Controls.BocListImplementation.Rendering
 {
-  public class StubColumnRenderer : BocColumnRendererBase<StubColumnDefinition>
+  public class StubColumnRenderer : IBocColumnRenderer
   {
     public StubColumnRenderer (IResourceUrlFactory resourceUrlFactory)
-        : base(resourceUrlFactory, Remotion.Web.UI.Controls.Rendering.RenderingFeatures.Default, new BocListCssClassDefinition(), new FakeFallbackNavigationUrlProvider())
     {
     }
 
-    protected override void RenderTitleCell (BocColumnRenderingContext<StubColumnDefinition> renderingContext, in BocTitleCellRenderArguments arguments)
+    public bool IsNull => false;
+
+    void IBocColumnRenderer.RenderTitleCell (BocColumnRenderingContext renderingContext, in BocTitleCellRenderArguments arguments)
     {
       renderingContext.Writer.AddAttribute("arguments-CellID", arguments.CellID);
       renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Th);
       renderingContext.Writer.RenderEndTag();
     }
 
-    protected override void RenderDataCell (BocColumnRenderingContext<StubColumnDefinition> renderingContext, in BocDataCellRenderArguments arguments)
+    void IBocColumnRenderer.RenderDataCell (BocColumnRenderingContext renderingContext, in BocDataCellRenderArguments arguments)
     {
       renderingContext.Writer.AddAttribute("arguments-CellID", arguments.CellID ?? "null");
       renderingContext.Writer.AddAttribute("arguments-HeaderIDs", arguments.HeaderIDs.Any() ? string.Join(", ", arguments.HeaderIDs) : "empty");
       renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Td);
       renderingContext.Writer.RenderEndTag();
     }
 
-    protected override void RenderCellContents (BocColumnRenderingContext<StubColumnDefinition> renderingContext, in BocDataCellRenderArguments arguments)
+    public void RenderDataColumnDeclaration (BocColumnRenderingContext renderingContext, bool isTextXml)
+    {
+      renderingContext.Writer.WriteBeginTag("col");
+      if (isTextXml)
+        renderingContext.Writer.Write(" />");
+      else
+        renderingContext.Writer.Write(">");
```

Comment:
The `else` part is rendering invalid XML tag as the `col` element is never closed. In this case we never need the `else` to begin with. If I am not mistaken the test should pass by just rendering the `col` element:

```suggestion
      enderingContext.Writer.Write(" />");
```